migrations/20210519031304-add-category-references.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addConstraint("Categories", {
+      fields: ["UserId"],
+      type: "FOREIGN KEY",
+      references: {
+        table: "Users",
+        field: "id",
+      },
+    });
+    await queryInterface.addConstraint("Categories", {
```

Comment:
It is incorrect to add a UNIQUE constraint, because that would make User/Category ONE-TO-ONE